api/src/libs/migrationUtils.ts

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { BusinessesDataClient, UserDataClient } from "@domain/types";
+import { Business, CURRENT_VERSION, UserData } from "@shared/userData";
+
+export async function processBusinessesMigration(
+  userData: UserData,
+  businessesData: Record<string, Business>,
```

Comment:
In both instances, `migrateBusinessesFromUsers` and `usersBusinessesMigration`, `businessesData` is pulled from `userData`, since that is the case, I think we can remove the parameter.